app/src/main/res/values/strings.xml

```diff
@@ -468,7 +468,8 @@
     <string name="wallpaper_tap_to_change_switch_label">Tap logo to change wallpaper</string>
     <!-- Description for switch which toggles the "tap-to-switch" behavior on home screen logo -->
     <string name="wallpaper_tap_to_change_switch_description">Cycle through and update image without leaving the homepage.</string>
-    <!-- Content description for switch which toggles the "tap-to-switch" behavior on home screen logo -->
+    <!-- This is the accessibility content description for the wallpapers functionality. Users are
+    able to tap on the app logo in the home screen and can switch to different wallpapers by tapping. -->
     <string name="wallpaper_logo_content_description">Firefox logo - change the wallpaper, button</string>
```

Comment:
I am wondering why we have `, ` before the button here?

Reply:
We got the text from product, you could read more here https://mozilla-hub.atlassian.net/browse/FNXV2-19315

Reply:
Should we remove it?

Reply:
The updated comment looks good to me.
FWIW, concerning the comma, I think I've seen this format used in other accessibility strings.

Reply:
Thanks for the clarification @Delphine!